build/xxHash/Makefile

```diff
@@ -0,0 +1,83 @@
+
+ROOT=../..
+include $(ROOT)/deps/readies/mk/main
+
+define HELPTEXT
+make build    # configure and compile
+make clean    # clean generated sbinaries
+  ALL=1       # remote entire binary directory
+endef
+
+MK_ALL_TARGETS=build
+
+#----------------------------------------------------------------------------------------------
+
+BINDIR=$(BINROOT)/xxHash
+SRCDIR=$(ROOT)/deps/xxHash
+
+TARGET=$(BINDIR)/libxxhash.a
+
+#----------------------------------------------------------------------------------------------
+
+MK_CUSTOM_CLEAN=1
+
+include $(MK)/defs
+
+CC=gcc
```

Comment:
why need to set this to gcc?

Reply:
good catch. not needed and will break with CLANG=1.